### PR TITLE
Add missing deps and other minor fixups to get CI green

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: packages/labs/gen-manifest/package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
       "dependencies": [
         "../../tests:build",
         "../analyzer:build",
-        "../gen-utils:build"
+        "../gen-utils:build",
+        "../../lit-element:build"
```
Comment (justinfagnani) [inline packages/labs/gen-manifest/package.json:23]: I don't see that anything in this package imports lit-element... why was this needed?

File: packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ test('basic wrapper generation', async () => {
   // Pack the generated package here, as `test-output` package.json will
   // reference the generated tarball here by filename; `test-output:installSelf`
   // depends on these tests run by `test:gen`.
-  await packPackage(outputPackage);
+  console.log(await packPackage(outputPackage));
```
Comment (justinfagnani) [inline packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts:67]: remove?

File: package.json [hunk 3]
```diff
@@ -282,7 +282,7 @@
     "rollup-plugin-terser": "^7.0.2",
     "typescript": "~5.2.0",
     "uvu": "^0.5.6",
-    "wireit": "^0.14.0"
+    "wireit": "^0.9.5"
```
Comment (AndrewJakubowicz) [inline package.json:285]: Why downgrade wireit?

File: packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts (hunk 2 above)
Comment (rictic) [reply packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts:67]: +1, done

**Review by AndrewJakubowicz — APPROVED** [review]

**Review by augustjk — APPROVED** [review]

File: packages/labs/gen-manifest/package.json (hunk 1 above)
Comment (rictic) [reply packages/labs/gen-manifest/package.json]: We're building a manifest with types for an example that imports lit, so we need the types for lit

File: packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts (hunk 2 above)
Comment (rictic) [reply packages/labs/gen-wrapper-react/src/test-gen/generate_test.ts]: Ah, yes, done

File: package.json (hunk 3 above)
Comment (rictic) [reply package.json]: Was wondering if that would fix, it did not
